src/Arachne.cc

```diff
@@ -561,10 +572,11 @@ dispatch() {
                 }
                 void** saved = &core.loadedContext->sp;
                 core.loadedContext = targetContext;
+                dispatchTimeTracker.flush();
```

Comment:
Add some comments here? e.g., need to call flush() here before swapcontext(), otherwise, Arachne will leak idle cycles.